extensions/reclaim-ai/src/task-form.tsx

```diff
@@ -217,7 +237,7 @@ export default (props: Props) => {
         onChange={setSnooze}
         type={Form.DatePicker.Type.DateTime}
         id="snoozeUntil"
-        title="Starting"
+        title="Start after"
```

Comment:
```suggestion
        title="Start After"
```
Tiles should be Upper camel cased, we try to follow Apple Style Guide - could you check the file, it looks like t here is more of them 🙂

Reply:
Updated. I think I found them all.